index.html

```diff
@@ -203,6 +204,20 @@ <h1 class="largest-heading">Rescue<span>a pet</span></h1>
       </div>
      </div>
     </section>
+    <section class="need-a-sweet-heart-wrapper">
+      <div class="need-a-sweet-heart-box">
+      <h1 class="largest-heading">Need a Sweet Heart?</h1>
+       <p class="small-description">
+         <span>Whether or not you plan on competing in dog sports and events,</span>
+         <span>it’s critical that all dogs receive basic training, from how to walk </span>
+         <span>well on leash to being comfortable with strangers</span>
+        </p>
+        <div class="need-a-sweet-heart-box-text-area">
+        <label><input class="need-a-sweet-heart-text-area" type="text" name="imie" placeholder="Enter your name"></label>
```

Comment:
stosuj angielskie nazwy
name="name"

https://www.w3schools.com/tags/tag_label.asp

label for="xxx"

Reply:
Poprawione

Reply:
Niepoprawione

Reply:
Poprawiłem lecz nie wiem czy zrobiłem dobrze nie jestem pewien.